### ISSUE TITLE
Add Azure Status to Monitor

### ISSUE THREAD

**Issue by kaftw** [instatushq/out#16]
There are a lot of other cloud providers listed, but I didn't see Microsoft's Azure cloud in the list.  Since I use it, it'd be great to have it in the app.

**Comment by alisalahio** [comment]
Azure is important, but their status page is custom.
It would be awesome if someone would contribute and implement it.

Here is how [stts](https://github.com/inket/stts) implements it:
[https://github.com/inket/stts/tree/master/stts/Services/Super](https://github.com/inket/stts/tree/master/stts/Services/Super)

**Comment by kaftw** [comment]
Yeah, I was looking into their RSS feed to see if that could be used, but I haven't had the time to dig too far into it: https://azurestatuscdn.azureedge.net/en-us/status/feed/.  From what I can tell, you could assume that there are no issues if there's nothing in the feed, which isn't really the best way to determine an outage.  😅

**Comment by alisalahio** [comment]
Yeah seems like this is a much simpler way that what stts is doing.
We could monitor this feed so we can check changes that happen when they're down 😂
